src/api/parser/package.json

```diff
@@ -1,28 +1,39 @@
 {
-  "name": "parser",
+  "name": "@senecacdot/parser-service",
+  "private": true,
   "version": "1.0.0",
-  "description": "A service for parsing posts",
+  "description": "A service for parsing feeds",
   "scripts": {
-    "start": "node src/server.js"
+    "dev": "env-cmd -f env.local node src/index.js",
+    "start": "node src/index.js"
   },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/Seneca-CDOT/telescope.git"
-  },
-  "author": "",
+  "repository": "Seneca-CDOT/telescope",
   "license": "BSD-2-Clause",
   "bugs": {
     "url": "https://github.com/Seneca-CDOT/telescope/issues"
   },
   "homepage": "https://github.com/Seneca-CDOT/telescope#readme",
   "dependencies": {
-    "@elastic/elasticsearch": "7.16.0",
-    "@elastic/elasticsearch-mock": "0.3.1",
+    "@bull-board/api": "3.7.0",
+    "@bull-board/express": "3.7.0",
     "@senecacdot/satellite": "latest",
-    "bull": "3.29.3",
-    "bull-board": "2.1.3",
-    "jsdom": "18.1.1",
-    "normalize-url": "6.1.0",
-    "rss-parser": "3.12.0"
+    "bull": "3.22.0",
+    "clean-whitespace": "0.1.2",
+    "highlight.js": "11.3.1",
+    "jsdom": "18.0.1",
+    "linkify-html": "3.0.5",
+    "linkifyjs": "3.0.5",
+    "normalize-url": "6.0.1",
+    "rss-parser": "3.12.0",
+    "sanitize-html": "2.5.3",
+    "set-interval-async": "2.0.3"
```

Comment:
Can we get rid of this and use https://nodejs.org/api/timers.html#timerspromisessetintervaldelay-value-options?